resources/authgear/templates/en/web/authflowv2/__password_strength_meter.html

```diff
@@ -1,16 +1,19 @@
 {{ define "authflowv2/__password_strength_meter.html" }}
 <meter
-  class="{{ $.Classname }} password-strength-meter w-full"
+  class="{{ $.Classname }} password-strength-meter"
   min="1"
   max="5"
+  data-controller="password-strength-meter"
   data-password-policy-target="currentMeter"
+  data-password-strength-meter-target="currentMeter"
```

Comment:
We can just use `this.element` instead of adding a duplicate target. See `mirrorbutton.ts` as an example.